cluster-autoscaler/expander/price/price.go

```diff
@@ -39,6 +39,7 @@ import (
 // **********
 
 type priceBased struct {
+	cloudProvider         cloudprovider.CloudProvider
```

Comment:
We probably don't need both cloudProvider and pricingModel (which can be obtained from cloudProvider) here.

Disregard this comment if you choose to follow the suggestion with GpuInfo struct, of course.

Reply:
True. I will clean it up.